js/react_native/android/build.gradle

```
@@ -20,6 +20,24 @@ def getExtOrIntegerDefault(name) {
   return rootProject.ext.has(name) ? rootProject.ext.get(name) : (project.properties['OnnxruntimeModule_' + name]).toInteger()
 }
 
+def checkIfOrtExtensionsEnabled() {
+  // locate user's project dir
+  def reactnativeRootDir = project.rootDir.parentFile
+  // get package.json file in root directory
+  def packageJsonFile = new File(reactnativeRootDir, 'package.json')
+  // read field 'ortExtensionsEnabled'
+  if (packageJsonFile.exists()) {
+    def packageJsonContents = packageJsonFile.getText()
+    def packageJson = new groovy.json.JsonSlurper().parseText(packageJsonContents)
+    return packageJson.ortExtensionsEnabled
```

we should probably be more explicit about what we accept. Groovy and Ruby (podspec) may handle boolean conversion differently and this should be consistent.

e.g., we could expect the string "true"
```suggestion
    return packageJson.ortExtensionsEnabled == "true"
```